sites/public/src/lib/helpers.tsx

```diff
@@ -133,75 +133,79 @@ export const getListingApplicationStatus = (listing: Listing): StatusBarType =>
   }
 }
 
-export const getListings = (listings: Listing[]) => {
-  const unitSummariesHeaders = {
-    unitType: "t.unitType",
-    minimumIncome: "t.minimumIncome",
-    rent: "t.rent",
-  }
-
-  const generateTableSubHeader = (listing) => {
-    if (listing.reviewOrderType !== ListingReviewOrder.waitlist) {
-      return {
-        content: t("listings.availableUnits"),
-        styleType: AppearanceStyleType.success,
-        isPillType: true,
-      }
-    } else if (listing.reviewOrderType === ListingReviewOrder.waitlist) {
-      return {
-        content: t("listings.waitlist.open"),
-        styleType: AppearanceStyleType.primary,
-        isPillType: true,
-      }
+const generateTableSubHeader = (listing) => {
+  if (listing.reviewOrderType !== ListingReviewOrder.waitlist) {
+    return {
+      content: t("listings.availableUnits"),
+      styleType: AppearanceStyleType.success,
+      isPillType: true,
+    }
+  } else if (listing.reviewOrderType === ListingReviewOrder.waitlist) {
+    return {
+      content: t("listings.waitlist.open"),
+      styleType: AppearanceStyleType.primary,
+      isPillType: true,
     }
-    return null
   }
+  return null
+}
+
+const unitSummariesHeaders = {
+  unitType: "t.unitType",
+  minimumIncome: "t.minimumIncome",
+  rent: "t.rent",
+}
 
+export const getListings = (listings: Listing[]) => {
   return listings.map((listing: Listing, index: number) => {
-    const uri = getListingUrl(listing)
-    const displayIndex: string = (index + 1).toString()
-    return (
-      <ListingCard
-        key={index}
-        imageCardProps={{
-          imageUrl:
-            imageUrlFromListing(listing, parseInt(process.env.listingPhotoSize || "1302")) || "",
-          tags: listing.reservedCommunityType
-            ? [
-                {
-                  text: t(`listings.reservedCommunityTypes.${listing.reservedCommunityType.name}`),
-                },
-              ]
-            : undefined,
-          statuses: [getListingApplicationStatus(listing)],
-          description: listing.name,
-        }}
-        tableProps={{
-          headers: unitSummariesHeaders,
-          data: getListingTableData(listing.unitsSummarized, listing.reviewOrderType),
-          responsiveCollapse: true,
-          cellClassName: "px-5 py-3",
-        }}
-        footerButtons={[
-          {
-            text: t("t.seeDetails"),
-            href: uri,
-            ariaHidden: true,
-          },
-        ]}
-        contentProps={{
-          contentHeader: {
-            content: displayIndex + ". " + listing.name,
-            href: uri,
-          },
-          contentSubheader: { content: getListingCardSubtitle(listing.buildingAddress) },
-          tableHeader: generateTableSubHeader(listing),
-        }}
-      />
-    )
+    return getListingCard(listing, index)
   })
 }
 
+export const getListingCard = (listing: Listing, index: number) => {
```

Comment:
There will 1000% be merge conflicts with #104 but I'll handle, feel free to merge first!

Reply:
Thank you! Let me know if I can help